tweets/first-test.tweet

```diff
@@ -0,0 +1,7 @@
+We are thrilled to be testing a new way to tweet polls directly from Github to expand participation in decision-making. 	
+@gr2m added a polling feature to twitter together https://github.com/gr2m/twitter-together/ to make this possible.	
+
+() This is exciting!	
+() :-D	
+() !!!	
+() ¯\_(ツ)_/¯
```

Comment:
Could you please start a new PR, but add spaces between the parentheses? 

```
( ) This is exciting!	
( ) :-D	
( ) !!!	
( ) ¯\_(ツ)_/¯
```

Don't merge it, the review should be enough to check if that was the problem